packages/design-system/src/components/InlineError/InlineError.tsx

```diff
@@ -13,7 +13,7 @@ interface InlineErrorProps {
   inversed?: boolean;
 }
 
-function InlineError({ children, className, id, inversed }: InlineErrorProps): React.ReactElement {
+export function InlineError({ children, className, id, inversed }: InlineErrorProps): React.ReactElement {
```

Comment:
@line47 Can you try now with this change? 

For me, with the create-react-app-typescript, it wasn't pulling in the InlineError export with `export * from './InlineError/InlineError';` in `index.d.ts` - I either had to change that line to `export { default as InlineError}` or put this export above.